src/KingSystem/Resource/resLoadRequest.h

```diff
@@ -3,7 +3,9 @@
 #include <basis/seadTypes.h>
 #include <prim/seadRuntimeTypeInfo.h>
 #include <prim/seadSafeString.h>
+#include <resource/seadResource.h>
 #include "KingSystem/Utils/Types.h"
+#include "KingSystem/Resource/resHandle.h"
```

Comment:
This change should be reverted -- res::Handle is forward declared to reduce compile times